packages/fluentui/docs/src/examples/components/Avatar/Variations/AvatarExampleSize.shorthand.tsx

```diff
@@ -19,15 +19,26 @@ const AvatarExampleSizeShorthand = () => (
       <React.Fragment key={size}>
         <strong>{size}</strong>
         <div>
-          <Avatar size={size} image="public/images/avatar/small/matt.jpg" status={statusProps} />
```

Comment:
The assets are still accessible using the relative path - what will stop me from using the relative path and mix the two approaches?

Reply:
I fixed this in a19ab2a, assets are no longer accessible via `/public`